internal/console/kics.go

```diff
@@ -2,35 +2,124 @@ package console
 
 import (
 	"context"
+	"errors"
+	"io"
 	"os"
 	"time"
 
+	consoleHelpers "github.com/Checkmarx/kics/internal/console/helpers"
+	"github.com/Checkmarx/kics/internal/constants"
 	"github.com/getsentry/sentry-go"
+	"github.com/gookit/color"
+	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
 )
 
-var ctx = context.Background()
-
 const (
 	scanID   = "console"
 	timeMult = 2
 )
 
-var rootCmd = &cobra.Command{
-	Use:   "kics",
-	Short: "Keeping Infrastructure as Code Secure",
-}
+var (
+	ctx = context.Background()
+
+	verbose  bool
+	logFile  bool
+	logLevel string
+	noColor  bool
+	silent   bool
+
+	warnings = make(map[string]bool)
+
+	rootCmd = &cobra.Command{
+		Use:   "kics",
+		Short: constants.Fullname,
+	}
+)
 
-func initialize() {
+func initialize() error {
 	rootCmd.AddCommand(versionCmd)
 	rootCmd.AddCommand(generateIDCmd)
 	rootCmd.AddCommand(scanCmd)
+	rootCmd.AddCommand(listPlatformsCmd)
+	rootCmd.PersistentFlags().BoolVarP(&logFile, "log-file", "l", false, "writes log messages to info.log")
+	rootCmd.PersistentFlags().StringVarP(&logLevel, "log-level", "", "INFO", "determines log level (TRACE,DEBUG,INFO,WARN,ERROR,FATAL)")
+	rootCmd.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "write logs to stdout too (mutually exclusive with silent)")
+	rootCmd.PersistentFlags().BoolVarP(&silent, "silent", "s", false, "silence stdout messages (mutually exclusive with verbose)")
+	rootCmd.PersistentFlags().BoolVarP(&noColor, "no-color", "", false, "disable CLI color output")
+
+	if err := viper.BindPFlags(rootCmd.PersistentFlags()); err != nil {
+		return err
+	}
 
 	initScanCmd()
 	if insertScanCmd() {
+		warnings["DEPRECATION WARNING: for future versions use 'kics scan'"] = true
```

Comment:
I think we could create a deprecation warning function, so the deprecation message would be standardized

Reply:
we can improve this on a new feature and a separate PR